Song/SongWriter.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Song
+{
+    public class SongWriter
+    {
+        public string Sing(List<string> animals, List<string> animalLines)
+        {
+            VerseWriter verserWriter = new VerseWriter();
```

Comment:
Every time you sing a song you will re-initialize verseWriter. Inneficient. In 99% of cases, this is expected to be done through ctor.